NOTICE

```diff
@@ -28,8 +22,9 @@ See bundled license files for details.
 This project bundles the following dependencies under SIL OFL 1.1 license (https://opensource.org/licenses/OFL-1.1).
 See bundled license files for details.
 
-- font-awesome:4.5.0 (Font) (http://fortawesome.github.io/Font-Awesome/) - Created by Dave Gandy
-    -> fonts in "docs/page/font-awesome/fonts"
+- font-awesome:4.6.3 (Font) (https://fontawesome.com/) - Created by Dave Gandy
+    -> css in "docs/static/font-awesome/css"
```

Comment:
css files are licensed under MIT

Reply:
Thanks for pointing out. I will update the css file in the MIT part. I think the version of css is 4.6.3. Is there a version for the fonts?